build/docker-deploy.sh

```diff
@@ -11,9 +11,9 @@ if [[ "${PUSH:-}" == "true" ]]; then
     base64 -d >~/.docker/config.json <<<"$OAO_DOCKER_AUTH"
 
     ./build/version.sh
-    docker tag dynatrace/dynatrace-gcp-function:latest dynatrace/dynatrace-gcp-function:$TAG
+    docker tag dynatrace/dynatrace-gcp-function:v2-latest dynatrace/dynatrace-gcp-function:$TAG
```

Comment:
consider to not change it to `v1-latest` because we are moving from `0.x` to `1.x`

Reply:
Shouldn't it be more like ```dynatrace/dynatrace-gcp-function/v1:latest``` ?

Reply:
No, image name should not change. It's just the tag change from "latest" to v1-latest"